src/auth/auth.controller.ts

```diff
@@ -38,4 +46,23 @@ export class AuthController {
   async register(@Body() userDto: CreateUserDto) {
     return this.authService.register(userDto);
   }
+
+  @Post("login")
+  @UseGuards(LocalAuthGuard)
+  @ApiOperation({
+    description: "Will return token when the account information is correct.",
+    summary: "login with our account",
+  })
+  @ApiCreatedResponse({
+    description: "Success with generated token",
+    type: GenerateTokenResponse,
+  })
+  @ApiUnauthorizedResponse({
+    description: "Account information error",
+    type: UserLoginUnauthorizedError,
+  })
```

Comment:
Will be Unauthorized or Forbidden?

Reply:
Revised.
[refactor: revised auth login api response description](https://github.com/Cophr/backend/pull/10/commits/adbdd009ce141e39836353afdf6b9e37d27beb57)